js/app.js

```diff
@@ -0,0 +1,10 @@
+$('#login').on('click', function(e) {
```

Comment:
Your validations don't show/work because you have the event handler hanging on to the button as opposed to the form. You'd want to do something like:

```javascript
$("form").on("submit", function (e) { // ... cont
```